src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -52,6 +53,8 @@
  */
 abstract class AbstractCategoryType extends TranslatorAwareType
 {
+    public const MAX_TITLE_LENGTH = 128;
```

Comment:
This should be placed in a dedicated `CategorySettings` class so that it can be used in other code without relying on the form type itself